WordPress/src/main/java/org/wordpress/android/ui/posts/photochooser/PhotoChooserFragment.java

```diff
@@ -66,49 +73,35 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mBottomBar.findViewById(R.id.icon_camera).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                handleIconClicked(PhotoChooserIcon.ANDROID_CAMERA);
+                mListener.onPhotoChooserIconClicked(PhotoChooserIcon.ANDROID_CAMERA);
```

Comment:
I know you left this here as a trap ;) (jk)
Is there a possibility `mListener` is null here? I can see this is populated [here](https://github.com/wordpress-mobile/WordPress-Android/pull/5277/files#diff-0c023198108318246102abd97075dba2R101) in `onAttach` and it seems natural there's this pre-condition (that the fragment _has_  to be attached) in order for a click event to ever happen, but I'd add the null check just in case.